src/olympia/reviewers/models.py

```diff
@@ -1178,13 +1178,19 @@ def check_should_be_delayed(cls, version):
         # first listed version - should have their auto-approval delayed for
         # 24 hours to give us time to catch spam. Langpacks are exempted from
         # this since they are submitted as part of Firefox release process.
-        is_langpack = version.addon.type == amo.ADDON_LPAPP
+        addon = version.addon
+        is_langpack = addon.type == amo.ADDON_LPAPP
         now = datetime.now()
-        nomination = version.nomination or version.addon.created
+        nomination = version.nomination or addon.created
+        try:
+            content_review = addon.addonapprovalscounter.last_content_review
+        except AddonApprovalsCounter.DoesNotExist:
+            content_review = None
         return (
             not is_langpack and
             version.addon.status == amo.STATUS_NOMINATED and
-            now - nomination < timedelta(hours=24))
+            now - nomination < timedelta(hours=24) and
+            content_review is None)
```

Comment:
perhaps we should check the content_review was performed after the nomination date.  Otherwise there is an edge-case where an add-on was approved and content reviewed previously; all it's versions were disabled so went back to incomplete; and a new version was uploaded so the add-on becomes nominated again - if that happened we probably would want to treat it like a new add-on... (right?  I'm not certain on this)

Reply:
mmm I would say no but I'm not certain either. IMHO the main purpose of the delay is to counter bad actors submitting low-effort malware that pretend to be flash player or adblocker or whatever and that would appear in the recently updated add-ons without the delay.

If an add-on has been approved & content reviewed once, that means it passed the delay once already, so I'm not sure it needs to go through it again (if a reviewer rejected it and all its versions the whole add-on would likely be disabled)

Reply:
cc @wagnerand for input

Reply:
I think we can ignore that case. Ideally we should reset content-approved whenever name, summary, description, screenshots etc change.